provider/trips.json

```diff
@@ -0,0 +1,191 @@
+      "$ref": "common.json#/definitions/version"
+    },
+    "data": {
+      "$id": "#/properties/data",
+      "type": "object",
+      "title": "The page of data",
+      "required": [],
+      "properties": {
+        "trips": {
+          "$id": "#/properties/data/properties/trips",
+          "type": "array",
+          "title": "The Trips Schema",
+          "items": {
+            "$id": "#/properties/data/properties/trips/items",
+            "type": "object",
+            "title": "The Trip Schema",
+            "required": [
+              "provider_name",
+              "provider_id",
+              "device_id",
+              "vehicle_id",
+              "vehicle_type",
+              "propulsion_type",
+              "trip_id",
+              "trip_duration",
+              "trip_distance",
+              "route",
+              "accuracy",
+              "start_time",
+              "end_time"
+            ],
+            "properties": {
+              "provider_name": {
+                "$id": "#/properties/data/properties/trips/items/properties/provider_name",
+                "type": "string",
+                "description": "The public-facing name of the Provider",
+                "examples": [
+                  "Provider Name"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "provider_id": {
+                "$id": "#/properties/data/properties/trips/items/properties/provider_id",
+                "description": "The UUID for the Provider, unique within MDS",
+                "$ref": "common.json#/definitions/uuid"
+              },
+              "device_id": {
+                "$id": "#/properties/data/properties/trips/items/properties/device_id",
+                "$ref": "common.json#/definitions/uuid",
+                "description": "A unique device ID in UUID format"
+              },
+              "vehicle_id": {
+                "$id": "#/properties/data/properties/trips/items/properties/vehicle_id",
+                "type": "string",
+                "description": "The Vehicle Identification Number visible on the vehicle itself",
+                "default": "",
+                "examples": [
+                  "ABC123"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "vehicle_type": {
+                "$id": "#/properties/data/properties/trips/items/properties/vehicle_type",
+                "type": "string",
+                "description": "The type of vehicle",
+                "enum": [
+                  "bicycle",
+                  "scooter"
+                ]
+              },
+              "propulsion_type": {
+                "$id": "#/properties/data/properties/trips/items/properties/propulsion_type",
+                "type": "array",
+                "description": "The type of propulsion; allows multiple values",
+                "items": {
+                  "type": "string",
+                  "enum": [
```

Comment:
Same as above, `"electric_assist"` was added to this list. 

Is there a way to refactor this (and `vehicle_type`) out into `common.json`?

Reply:
Yep, happy to try to refactor that into the common definitions.